src/js/components/ChooseProjectModal.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { useState } from 'react';
+import { Tabs, List, Avatar, Modal, Button, Space, Typography } from 'antd';
+import { useAppSelector } from '@/hooks';
+import { Link, useLocation } from 'react-router-dom';
+
+const ChooseProjectModal: React.FC<ChooseProjectModalProps> = ({ isModalOpen, setIsModalOpen }) => {
```

Comment:
React.FC is not ideal as a type: see https://github.com/facebook/create-react-app/pull/8177